usvm-dataflow-ts/src/main/kotlin/org/usvm/dataflow/ts/infer/annotation/ValueTypeAnnotator.kt

```diff
@@ -31,16 +31,22 @@ import org.jacodb.ets.model.EtsField
 import org.jacodb.ets.model.EtsScene
 import org.usvm.dataflow.ts.infer.AccessPathBase
 import org.usvm.dataflow.ts.infer.EtsTypeFact
-import org.usvm.dataflow.ts.infer.dto.toType
+import org.usvm.dataflow.ts.infer.toType
 
 class ValueTypeAnnotator(
```

Comment:
let's remove it

Reply:
Let's keep it for now, it might be needed soon, and we already cleaned it up to (probably) be more convenient to use